### PR TITLE
Add updating status api client

### PR DESCRIPTION
**Pull request by yuichi0301**
Please review!

### CODE REVIEW

File: src/main/java/com/nulabinc/backlog4j/api/option/UpdateStatusParams.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
+package com.nulabinc.backlog4j.api.option;
+
+import com.nulabinc.backlog4j.http.NameValuePair;
+
+/**
+ * Parameters for update status API.
+ *
+ * @author nulab-inc
+ */
+public class UpdateStatusParams extends PatchParams {
```
Comment (shomatan) [inline src/main/java/com/nulabinc/backlog4j/api/option/UpdateStatusParams.java]: Status color(param: color) should be changeable.
Please refer https://developer.nulab.com/docs/backlog/api/2/update-status/#form-parameters

File: src/main/java/com/nulabinc/backlog4j/api/option/UpdateStatusParams.java [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    private Object projectIdOrKey;
+    private Object statusId;
```
Comment (shomatan) [inline src/main/java/com/nulabinc/backlog4j/api/option/UpdateStatusParams.java]: Status color(param: color) should be changeable.
Please refer https://developer.nulab.com/docs/backlog/api/2/update-status/#form-parameters

Comment (yuichi0301) [reply src/main/java/com/nulabinc/backlog4j/api/option/UpdateStatusParams.java]: Oh..I missed it.

Comment (yuichi0301) [reply src/main/java/com/nulabinc/backlog4j/api/option/UpdateStatusParams.java]: I fixed it. Please re-review!
@shomatan

**Comment by shomatan** [comment]
@yuichi0301 LGTM~♬

**Pull request merged**